<?xml version="1.0"?>
<launch>

    <group ns="vehicle">

        <!-- Remap topics -->
        <remap from="current_pose"      to="/localization/current_pose" />
        <remap from="current_velocity"  to="/localization/current_velocity" />
        <remap from="vehicle_cmd"       to="/control/vehicle_cmd" />

        <!-- Bicycle_model simulator -->
        <node pkg="autoware_mini" type="bicycle_simulation.py" name="bicycle_simulation" output="screen" required="true" />
    
    </group>


</launch>